tests/onnx/quantization/test_classification_models_graph.py

```diff
@@ -48,3 +50,17 @@ def test_min_max_quantization_graph(tmp_path, mocker, model_to_test, model):
     original_model = onnx.load(onnx_model_path)
     quantized_model = min_max_quantize_model(model_to_test.input_shape, original_model)
     compare_nncf_graph(quantized_model, model_to_test.path_ref_graph)
+
+
+ONNX_TEST_DATA = [ModelToTest('densenet-12', [1, 3, 224, 224])]
+
+
+@pytest.mark.parametrize(('model_to_test'), ONNX_TEST_DATA,
+                         ids=[model_to_test.model_name for model_to_test in ONNX_TEST_DATA])
+def test_min_max_quantization_graph_onnx_model(tmp_path, mocker, model_to_test):
```

Comment:
Why do we need this test now?

Reply:
Because densenet model has the pattern that I added, I want to test it by this test